faq.md

```diff
@@ -138,4 +174,25 @@ Cockpit uses PackageKit for the Software updates page. PackageKit does not curre
 More details at:
 - [Red Hat bug 1237014: PackageKit ignores exclude in /etc/dnf/dnf.conf](https://bugzilla.redhat.com/show_bug.cgi?id=1237014#c6)
 - [Cockpit bug #17955: Cockpit update is ignoring excludes in yum/dnf configurations](https://github.com/cockpit-project/cockpit/issues/17955)
-- [SysTutorials: How to exclude a package from a specific repository only in yum?](https://www.systutorials.com/how-to-exclude-a-package-from-a-specific-repository-only-in-yum/)
\ No newline at end of file
+- [SysTutorials: How to exclude a package from a specific repository only in yum?](https://www.systutorials.com/how-to-exclude-a-package-from-a-specific-repository-only-in-yum/)
+
+### Virtual machines
+
+#### Virtual machines not working as expected
+
+If a VM hangs during boot or you see an error message such as "No bootable device", it may be due to a configuration option in your computer's BIOS / EFI settings.
```

Comment:
Again I don't understand this. The VM boot isn't related to the host's BIOS/EFI settings at all. Perhaps the single exception is KVM support, but depending on how the VM is configured, that will either just make it very slow, or not start at all (if the VM requires KVM). I have no proposal how to update the text, as I don't know what it means. Do you have a link to the original issue/report? (as a FAQ, it should have been asked at least twice, I figure :grin: )

Reply:
This happens. Here's a recent issue: https://github.com/cockpit-project/cockpit-machines/issues/879

This is tied to a setting in BIOS/EFI and I've seen it myself before in the past too.

I always set that to enabled, but I have forgotten on a new system and have even seen it reset on a BIOS update (and ran into the not being able to start a VM issue as a result). Since the person who reported it and I have both experienced it (multiple times over the years), I think it's worth a mention.

Reply:
I also opened an issue @ https://github.com/cockpit-project/cockpit-machines/issues/891 to add a check for it. However, even once we implement something like this (if we can) then it still doesn't cover Enterprise / LTS versions of Cockpit, so I think we still would want to mention it in the FAQ.

Reply:
Thanks!